lib/circuitry/subscription_creator.rb

```diff
@@ -0,0 +1,57 @@
+require 'circuitry/services/sns'
+require 'circuitry/topic'
+
+module Circuitry
+  class SubscriptionCreator
+    include Services::SNS
+    include Services::SQS
+
+    attr_reader :queue
+    attr_reader :topics
+
+    def self.subscribe_all(queue, topics)
+      new(queue, topics).subscribe_all
+    end
+
+    def initialize(queue, topics)
```

Comment:
Should we have argument checking here like we have in some other classes in this gem?

```ruby
raise ArgumentError, 'topics must be an array' unless topics.is_a?(Array)
```